aws/track.yml

```diff
@@ -0,0 +1,77 @@
+slug: aws-cloud-account
+id: suhbhzjy3lxo
+type: track
+title: AWS Cloud Account
+teaser: Learn how to build tracks with an AWS account
+description: |-
+  Learn how to use a sandboxed AWS account in your Instruqt track.
+
+  **This track demonstrates how to:**
+    - Link an AWS cloud account to your sandbox
+    - How to use the cloud client Docker container
+    - Use the AWS CLI and UI from our learn environment
+    - Limit usage of the cloud account using IAM and SCP policies
+icon: https://cdn.instruqt.com/assets/templates/aws.png
+tags:
+- Cloud Account
+- Docker
+owner: templates
+developers:
+- arslan@instruqt.com
+- matthisk@instruqt.com
+private: false
+published: true
+challenges:
+- slug: aws-ec2-create
+  id: stswhtvyegcl
+  type: challenge
+  title: AWS EC2 instance creation
+  teaser: Every cloud starts from VM
+  notes:
+  - type: text
+    contents: |-
+      You're about to create an EC2 instance using AWS CLI.
+
+      Please wait while we provision the AWS account.
```

Comment:
> You're about to create an EC2 instance using AWS CLI.

I think this sentence could be rephrased with learning objectives.

`In this track, you will learn how to provision EC2 instances on AWS using the CLI.`

Something like this.

> Please wait while we provision the AWS account.

I think this message is redundant, as we're already indicating a loading state with our spinner.